mirascope/rag/types.py

```diff
@@ -22,7 +22,7 @@ class BaseEmbeddingResponse(BaseModel, Generic[ResponseT], ABC):
 
     @property
     @abstractmethod
-    def embedding(self) -> list[float]:
+    def embeddings(self) -> Optional[Union[list[list[float]], list[list[int]]]]:
```

Comment:
we're allowing int because cohere supports int?

Reply:
yes